### DISCUSSION TITLE
For you (beta version)

### DISCUSSION THREAD

**Discussion by kaiquekandykoga** [community/community#13253] [Other Feature Feedback, Questions, & Ideas]
I like the idea of having an easy way to check information about what is going on.

However I can see many people starring or creating repositories, and I do not want see these people. I am not even following these people. They just worked with me in the past and perhaps that is the reason I am seeing them?

I like GitHub, and I hope my feedback can help to improve this feature.

Thank you.

**Comment by krishpranav** [comment]
```For you``` is a great feature i just love it

**Reply by IMB11** [reply]
Its actually something I've been waiting for for a while! I love finding niche projects through the explore tab and For You just makes it so much better.

**Comment by jfullstackdev** [comment]
hello guys, there is a new update on the For You feed, you can now filter it, and for those who don't want it can simply disable it by unchecking all filters

![image](https://user-images.githubusercontent.com/47092464/182012117-0b19be9d-e849-462e-909e-d20d28915c1d.png)

**Reply by takpar11** [reply]
Can you teach me bro?

**Reply by petedegraw** [reply]
I don't use GitHub for social coding, I don't care about the options provided in this screenshot. I cannot imagine a use case where I'd want to see any of the options in the dropdown. I think a product designer should consult on what "dashboard" means and how it serves users. This current screenshot feels way off.

What would be useful to toggle in the dropdown would be seeing commits on projects I work on, PRs on projects I work on...

**Comment by Grezor** [comment]
Hello, 
I have encountered a problem with the checkbox. When I click on the text (example: stars), the checkbox is deactivated. But some when you click on the checkbox, it unchecks. But not all.  

Small information: it is necessary to click on the text, so that it unchecks the checkboxes. I had not understood very well at first.

![image](https://user-images.githubusercontent.com/38507456/219034552-c94ef85c-5fcd-47e9-ad6d-028ef80f4149.png)

**Comment by iamlorddop** [comment]
Hello, you need to uncheck the filter here https://github.com/
